name.abuchen.portfolio.junit/src/name/abuchen/portfolio/junit/MessagesTest.java

```diff
@@ -130,6 +131,11 @@ private void test(String bundleName)
 
                 String test = MessageFormat.format(value, (Object) null);
                 assertThat(test, is(notNullValue()));
+
+                assertFalse(value, value.contains("\uFFFD"));
+                assertFalse(value, value.contains("\uFFEF"));
+                assertFalse(value, value.contains("\uFFBF"));
+                assertFalse(value, value.contains("\uFFBD"));
```

Comment:
Why did you add this character?

It looks to me like a regular character: https://unicode-table.com/en/FFBD/
ﾽ Halfwidth Hangul Letter Phieuph